2019/media/playbackperfTest.js

```diff
@@ -25,7 +25,17 @@ var PlaybackperfTest = function() {
 
 var webkitPrefix = MediaSource.prototype.version.indexOf('webkit') >= 0;
 var tests = [];
-TestBase.timeout = 70000; // 70 sec to compensate for 0.25 playbackRate tests.
+var testTime = harnessConfig.testTime;
+
+if (harnessConfig.timeout == TestBase.timeout) {
```

Comment:
Hm, where gets `TestBase.timeout` initially set? I don't see that here and would assume that we always hit this if condition. Why not checking if `harnessConfig.timeout` is undefined?